api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -245,13 +245,23 @@
 								 baseTableName="encounter_diagnosis" baseColumnNames="patient_id"
 								 referencedTableName="patient" referencedColumnNames="patient_id" />
 	</changeSet>
-
 	<changeSet id="20180405131015-TRUNK-5333" author="alicerowan">
 		<preConditions onFail="MARK_RAN">
 			<columnExists tableName="concept_numeric" columnName="precise"/>
 		</preConditions>
 		<comment>Rename concept_numeric.precise to concept_numeric.allow_decimal</comment>
 		<renameColumn oldColumnName="precise" newColumnName="allow_decimal" tableName="concept_numeric"  columnDataType="BOOLEAN"/>
 	</changeSet>
-
+	<changeSet id="20180323-1000" author="Kelechi+iheanyi" dbms="mysql">
+		<preConditions onFail="MARK_RAN">
+			<tableExists tableName="condition" />
+		</preConditions>
+		<comment>
+			Migrate data from the conditions table to the new condition table
+		</comment>
+		<sql>
+			INSERT into `condition` (condition_id, previous_version, patient_id, clinical_status, condition_non_coded, onset_date, additional_detail, end_date, creator, date_created, voided, voided_by, date_voided, void_reason, uuid)
```

Comment:
This sql produces errors when run directly against my database.

Reply:
To run it directly against the database, it usually requires the db name used in conjunction with the table name. Example, if the database name is `dbname`, it would require `dbname.condition` instead of `condition`.

Reply:
Would this changeset work without errors, for some one who has never run the emrapi module?

Reply:
No. There would be the absence of the condition table.

Reply:
So are you planning to fix it? 😊

Reply:
I have modified the `preconditions` to continue even if it fails.

Reply:
That right thing to do is simply add a check for the existence of the table created by the emrapi module

Reply:
That is what the preConditions are supposed to do, I guess. 
I perceive I should check if `condition` and `conditions` tables exist. If it fails, I should mark the changeset as run.

@dkayiwa